combat-trainer.lic

```diff
@@ -213,6 +213,9 @@ class LootProcess
     @ritual_type = thanatology['ritual_type'].downcase
     echo("  @ritual_type: #{@ritual_type}") if $debug_mode_ct
 
+    @cycle_rituals = false
```

Comment:
I don't think you need a separate variable for this. You would not reassign `@ritual_type`, but instead pass it around as an argument.